src/App.tsx

```diff
@@ -17,18 +49,32 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                setSelectItem={setSelectItem}
+                query={query}
+                setQuery={(value: string) => setQuery(value)}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {loading && (
+                <Loader />
+              )}
+
+              {!loading && todos.length > 0 && (
```

Comment:
pls [see falsy values](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)
```suggestion
              {!loading && !!todos.length && (
```